src/frontend/src/CustomNodes/GenericNode/components/outputModal/components/switchOutputView/index.tsx

```diff
@@ -35,30 +38,66 @@ const SwitchOutputView: React.FC<SwitchOutputViewProps> = ({
   if (resultMessage?.raw) {
     resultMessage = resultMessage.raw;
   }
+
+  const resultMessageMemoized = useMemo(() => {
+    if (
+      typeof resultMessage === "string" &&
+      resultMessage.length > MAX_TEXT_LENGTH
+    ) {
+      resultMessage = `${resultMessage.substring(0, MAX_TEXT_LENGTH)}
+
+The text is too long. Displaying the first ${MAX_TEXT_LENGTH + 1} characters...`;
```

Comment:
It would better if we just added the `...` at the end. This phrase has too much info for the user